core/src/main/java/com/testaem/aem/core/models/impl/CustomFooterImpl.java

```diff
@@ -0,0 +1,21 @@
+package com.testaem.aem.core.models.impl;
+
+import com.testaem.aem.core.models.CustomFooter;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.annotations.DefaultInjectionStrategy;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.injectorspecific.ValueMapValue;
+
+@Model(adaptables = Resource.class, defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL, adapters = CustomFooter.class)
+public class CustomFooterImpl implements CustomFooter {
+
+    static final String RESOURCE_TYPE = "testaem/components/customfooter";
```

Comment:
This constant is not used now. You probably wanted to use it in `@Model`-s `resourceType` propertyu

Reply:
corrected in the last commit.